developer_setup/classic_ui_split.md

```diff
@@ -14,7 +14,7 @@ But if you do..
 1. `git clone git@github.com:YOU/manageiq-ui-classic`
 1. `cd manageiq-ui-classic`
 1. `git remote add upstream git@github.com:ManageIQ/manageiq-ui-classic.git`
-1. `ln -s ../../manageiq spec/`
+1. `ln -s ../manageiq spec/`
```

Comment:
I think it is correct and it needs to be this way.

Look at this excerpt from `man ln`.

Symbolic links can hold arbitrary text; if later resolved, a relative link is interpreted in relation to its parent directory.

In this case it is relative to `manageiq/spec/` dir not the current dir of `manageiq/`